src/Razor/src/Microsoft.AspNetCore.Razor.LanguageServer/EndpointContracts/ITextDocumentPositionParams.cs

```diff
@@ -1,13 +0,0 @@
-﻿// Copyright (c) .NET Foundation. All rights reserved.
-// Licensed under the MIT license. See License.txt in the project root for license information.
-
-using Microsoft.VisualStudio.LanguageServer.Protocol;
-
-namespace Microsoft.AspNetCore.Razor.LanguageServer.EndpointContracts;
-
-internal interface ITextDocumentPositionParams
```

Comment:
This is susprising. Was this just unused from a while ago?

Reply:
It is used, but [MS.VS.LS.Protocol created an equivelent](https://devdiv.visualstudio.com/DevDiv/_git/VSLanguageServerClient?path=/src/product/Protocol/LanguageServer.Protocol/ITextDocumentPositionParams.cs).

Reply:
Nice!